app/javascript/services/forest-data.js

```diff
@@ -39,16 +39,17 @@ const SQL_QUERIES = {
   faoEcoLive:
     'SELECT fao.country, fao.forempl, fao.femempl, fao.usdrev, fao.usdexp, fao.gdpusd2012, fao.totpop1000, fao.year FROM table_7_economics_livelihood as fao WHERE fao.year = 2000 or fao.year = 2005 or fao.year = 2010 or fao.year = 9999',
   nonGlobalDatasets:
-    "SELECT iso, polyname FROM data WHERE polyname IN ('plantations', 'mining', 'primary_forest', 'landmark', 'plantations__mining', 'plantations__landmark', 'primary_forest__mining', 'primary_forest__landmark') GROUP BY iso, polyname ORDER BY polyname, iso"
+    "SELECT iso, polyname FROM data WHERE polyname IN ('plantations', 'mining', 'primary_forest', 'landmark', 'plantations__mining', 'plantations__landmark', 'primary_forest__mining', 'primary_forest__landmark') GROUP BY iso, polyname ORDER BY polyname, iso",
+  globalLandCover: 'SELECT * FROM global_land_cover_adm2 WHERE {location}'
 };
 
 const getExtentYear = year =>
   (year === 2000 ? 'area_extent_2000' : 'area_extent');
 
 const getLocationQuery = (country, region, subRegion) =>
-  `${country ? `iso = '${country}' AND` : ''}${
-    region ? ` adm1 = ${region} AND` : ''
-  }${subRegion ? ` adm2 = ${subRegion} AND` : ''}`;
+  `${country ? `iso = '${country}'` : ''}${
```

Comment:
This is used by a lot of queries, and subsequently changing it has broken other widget fetches. See tree loss for Brazil widget. This needs to be reversed.

Reply:
After further review this is a special case compared to all other queries. In the global widget cases we sometimes don't want 'AND' after {location}. However in this case we need it separated from the sql. Tricky.

Reply:
Oh I didn't see any widget broken. Let me check again!
Anyway, we need to adjust this logic because print `AND` at the end of all cases doesn't have much logic because if the following conditions are not met, it breaks.

Reply:
For the case of global widgets fetches it is necesary, as we dont want to send a location so we need to remove the AND either at the beginning or the end.

Reply:
Fixed!